docs/team/zzthian.md

```diff
@@ -3,44 +3,42 @@ layout: page
+### Project: GREWZ
 
+GREWZ is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
 Given below are my contributions to the project.
 
-* **New Feature**: Added the ability to undo/redo previous commands.
-  * What it does: allows the user to undo all previous commands one at a time. Preceding undo commands can be reversed by using the redo command.
-  * Justification: This feature improves the product significantly because a user can make mistakes in commands and the app should provide a convenient way to rectify them.
-  * Highlights: This enhancement affects existing commands and commands to be added in future. It required an in-depth analysis of design alternatives. The implementation too was challenging as it required changes to existing commands.
-  * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
+* **New Feature**: Added the ability to add pictures into the application.
+  * What it does: allows the user to select a .jpg file and upload it into the application as a student picture.
+  * Justification: this feature improves the product significantly because it addresses the user's needs to help them recognise students easier.
+  * Highlights: This enhancement adds an image of the students in the UI. The implementation was challenging as it required understanding of JavaFX file chooser and its usage.
 
-* **New Feature**: Added a history command that allows the user to navigate to previous commands using up/down keys.
+* **New Feature**: Added the ability to create Assignments.
+  * What it does: allows the user to add Assignments into their task list. An assignment is a type of task that allows the user to keep track of students who have and have not submitted an assignment.
+  * Justification: this feature improves the product by enabling users to monitor their student's work progress.
+  * Highlights: Students can be added and removed from the assignment using the edit-task command. It required careful consideration over other design implementations, but due to time constraints a simple one was chosen for now.
 
-* **Code contributed**: [RepoSense link]()
+* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2022-09-16&tabOpen=true&tabType=authorship&tabAuthor=zzthian&tabRepo=AY2223S1-CS2103T-W12-4%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code&authorshipIsBinaryFileTypeChecked=false&authorshipIsIgnoredFilesChecked=false)
```

Comment:
I think this is not the correct link, I think it is [this](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=zzthian&breakdown=true&sort=groupTitle&sortWithin=title&since=2022-09-16&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other).

Reply:
Thanks!